.env.sample

```diff
@@ -29,3 +29,5 @@ SIDEKIQ_WEB_UI_PASSWORD='password'
 HMRC_INTERFACE_HOST='https://fake-laa-hmrc-interface-env'
 HMRC_INTERFACE_UID='TestHMRCInterfaceUID'
 HMRC_INTERFACE_SECRET='TestHMRCInterfaceSecret'
+
+ENVIRONMENT='environment'
```

Comment:
more idiomatic to use `ENV` or `HOST_ENV`  (inline with RAILS_ENV, RACK_ENV - we did this in CCCD. alternatively could use `HostEnv` to programmatically determine this, as in Apply or `RailsHost` as in CCCD (combinded with `ENV` envvar, for example.